.github/workflows/build_and_test.yml

```diff
@@ -9,13 +9,20 @@ on:
 jobs:
   build_and_test:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        testTask:
+          - pythonTest
+          - microPythonTest
     steps:
       - name: Check out
         uses: actions/checkout@v2
         with:
           fetch-depth: 0 # Fetch the whole git history, to be able to calculate some stats based on it.
       - name: Install Kotlin for scripting
         run: sudo snap install --classic kotlin
+      - name: Install MicroPython
+        run: sudo snap install micropython
```

Comment:
Ideally it should happen only for MicroPython, but in practice it takes only a handful of seconds so I consider it a fair simplification of the CI script.

Reply:
What about global warming? Looked like we could easily add `if` here 😅

Well, feel free to leave it as is actually. I'm just a perfectionist.

Reply:
Time spent on looking how to add an `if` to this YAML-based DSL exceeds the cost of leaving it here :P